prime-router/metadata/valuesets/sender-automation.valuesets

```diff
@@ -0,0 +1,281 @@
+# Sender Automation specific valueset to translate inbound values to RS accepted values
+#
```

Comment:
Obviously these are pretty non-standard, so I'm glad they are separated out.   I wonder if they are going to be useful mappings for all Experity sites, or useful only for GUC?

Reply:
These will be useful for all clients, not just Experity or GUC.  The problem with these EMRs and small hospital systems like CPSI and Meditech are that the AOEs are free-text eventually, we add enough to this valueset to account for most data entry.